README.md

```diff
@@ -1541,6 +1547,196 @@ support preserving stats between role reload
 (see [tarantool/metrics#334](https://github.com/tarantool/metrics/issues/334)),
 thus this feature will be unsupported for `metrics` driver.
 
+### Read view
+
+A read view is an in-memory snapshot of data on instances that isn’t affected by future data modifications. Read views allow you to retrieve data using the `read_view_object:select()` and `read_view_object:pairs()` operations.
+
+Read views can be used to make complex analytical queries. This reduces the load on the main database and improves RPS for a single Tarantool instance.
+
+Read views have the following limitations:
+
+  * Only the memtx engine is supported.
+  * Read view can be used starting from Tarantool Enterprise v2.11.0.
```

Comment:
Let's add one more crud-specific point.

* There is no clusterwide readview support. For a sharded cluster, we open a readview on each storage. Due to a cluster's distributed nature, it is not guaranteed that they will open simultaneously.